sechubingest.py

```diff
@@ -134,9 +134,14 @@ def _trim_asset(self, asset):
             if key in fields:
                 trimmed[key] = asset[key]
 
+        # Check if any aws key is set, if not dont process it and return None
+        aws_fields = {k: v for k, v in trimmed.items() if k.startswith('aws')}
+        if any(aws_fields.values()) is False:
+            return None
+
         # ensure all of the required fields are in the asset.
         for requirement in required:
-            if requirement not in trimmed.keys():
+            if requirement not in trimmed.keys() or asset[requirement] is None:
```

Comment:
Is this necessary?

Reply:
Yes, as all the `aws` related keys always available in the response of export API only if its AWS asset it will have data set otherwise those keys will have null values.. so this block never reject non aws data.
Simply put `if requirement not in trimmed.keys()` is never false